lib/trento/application/usecases/sap_systems/health_summary_service.ex

```diff
@@ -0,0 +1,107 @@
+defmodule Trento.Application.UseCases.SapSystems.HealthSummaryService do
```

Comment:
Maybe it could be just `Trento.SapSystems.HealthSummaryService`? Are we using `Application` and `UseCases` anywhere else?

(I know this is our first domain service or at least I guess, so feel free to yell at me)

Reply:
no you are right, this is how it should be but we didn't refactor the other modules yet, i guess we could use the "wrong" naming and then open an issue about the renaming